### PR TITLE
Convert Nightmare acceptance tests to Cypress

### PR DESCRIPTION
**Pull request by eessex**
Follow up to #4922 and addition of Cypress integration suite, this PR replicates nightmare specs for the following routes:
- `/` (home)
- `/artist/:id`
- `/artwork/:id`
- `/collect`
- `/:partner_id`
- `/tag/:id`
- `/terms`
- `/system/up`
- `/dev/blank` - never heard of this but sounds like used by Eigen.

Existing cypress tests for `/collections` and `collection/:id` have been updated to include all assertions from the nightmare suite.

Relates to https://artsyproduct.atlassian.net/browse/PLATFORM-2115

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The staging version of Metaphysics has breaking changes, this might not affect your build, but could mean your queries don't work the way you expect. See below for breaking changes:</td>
    </tr>
  </tbody>
</table>


- :warning: Enum value 'AVAILABILITY_DESC' was removed from enum 'ArtworkSorts'
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against fe83b48cb7d3ec3104cecd3fb6572b8a6dca8727
</p>

File: cypress/integration/artist.spec.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { artworkGridRenders } from "../helpers/artworkGridRenders"
+
+describe("/artist/:id", () => {
```
Comment (damassi) [inline cypress/integration/artist.spec.js]: 👍

File: cypress/integration/artist.spec.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  before(() => {
+    cy.visit("/artist/pablo-picasso")
+  })
```
Comment (anandaroop) [inline cypress/integration/artist.spec.js]: These `before` clauses will result in a full network request for each `it` test case, correct?

(Seems fine for now, but potentially something to optimize if the duration of the Cypress suite becomes a concern?)

File: cypress/integration/devBlank.spec.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+describe("/dev/blank", () => {
+  it("renders a blank page for Eigen", () => {
+    cy.visit("/dev/blank")
+    cy.contains("page intentionally left blank")
```
Comment (anandaroop) [inline cypress/integration/devBlank.spec.js]: John Cage energy

File: cypress/integration/partnerProfile.spec.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+      cy.visit("gagosian-gallery/shows")
+      cy.contains("Current Shows")
+    })
+    it("shows partner articles", () => {
+      cy.visit("gagosian-gallery/articles")
+      cy.contains("Articles")
+    })
+
+    xit("does not show contact information for non-active partner", () => {
+      // TODO: cy.visit(":partner_id/contact")
```
Comment (anandaroop) [inline cypress/integration/partnerProfile.spec.js]: Intentionally pended, yes?

File: cypress/integration/artist.spec.js (hunk 2 above)
Comment (eessex) [reply cypress/integration/artist.spec.js]: These will use one request at the beginning of the suite, all `it` clauses will be referring to the same request. (`beforeEach` would make a new request for each test).

File: cypress/integration/partnerProfile.spec.js (hunk 4 above)
Comment (eessex) [reply cypress/integration/partnerProfile.spec.js]: Yeah, I'm not sure what a good inactive partner to reference here is-- do you know one we could use?

File: cypress/integration/artist.spec.js (hunk 2 above)
Comment (anandaroop) [reply cypress/integration/artist.spec.js]: Ah perfect, thanks

File: cypress/integration/partnerProfile.spec.js (hunk 4 above)
Comment (anandaroop) [reply cypress/integration/partnerProfile.spec.js]: I've been trying a few different Looker queries but coming up empty for a good candidate.

Question tho: the test as written, would it actually distinguish between say a churned partner and a regular 404 for a non-existent partner? (I think that error message is the 404 one)

Comment (eessex) [reply cypress/integration/partnerProfile.spec.js]: The 404 is at the `/:partner_id/contact` route, we'd still be expecting the `/:partner_id` to 200 (I think? It was asserted without trying the parent route).

Comment (anandaroop) [reply cypress/integration/partnerProfile.spec.js]: Happy to merge and come back to this. (Tbh this maybe even seems kind of edge-casey for this test suite?)
